packages/text-field/test/text-field.test.js

```diff
@@ -50,101 +49,42 @@ describe('text-field', () => {
       });
 
       ['maxlength', 'minlength'].forEach((prop) => {
-        it(`should set numeric attribute ${prop}`, async () => {
+        it(`should delegate ${prop} attribute to the input`, async () => {
           await assertAttrCanBeSet(prop, 2);
         });
       });
 
       ['autocomplete'].forEach((prop) => {
-        it(`should set boolean attribute ${prop}`, async () => {
+        it(`should delegate ${prop} attribute to the input`, async () => {
           await assertAttrCanBeSet(prop, 'on');
         });
       });
 
       ['autocapitalize'].forEach((prop) => {
-        it(`should set boolean attribute ${prop}`, async () => {
+        it(`should delegate ${prop} attribute to the input`, async () => {
           await assertAttrCanBeSet(prop, 'none');
         });
       });
 
       ['autocomplete', 'autocorrect', 'readonly', 'required'].forEach((prop) => {
-        it(`should set boolean attribute ${prop}`, async () => {
+        it(`should delegate ${prop} attribute to the input`, async () => {
           await assertAttrCanBeSet(prop, true);
           await assertAttrCanBeSet(prop, false);
         });
       });
     });
 
-    describe('clear button', () => {
```

Comment:
Moved the default value test to this suite, all the rest are already covered there:

https://github.com/vaadin/web-components/blob/f67d18ca4f9a38e1798a513e99402916c0d6d148/packages/field-base/test/clear-button-mixin.test.js#L46